core/src/main/java/org/apache/sdap/mudrod/main/MudrodEngine.java

```diff
@@ -235,7 +237,7 @@ public void startMetaIngest() {
     LOG.info("Metadata has been ingested successfully.");
   }
 
-  public void startFullIngest() {
+  public void startFullIngest(MudrodEngine me) {
     DiscoveryEngineAbstract wd = new WeblogDiscoveryEngine(props, es, spark);
```

Comment:
EventIngester has a variable of MudrodEngine type since EventIngester extends MudrodAbstract class. I don't think you have to change function from startFullIngest() to startFullIngest(MudrodEngine me).